### PR TITLE
FOUNDATIONS: Group Create

### PR DESCRIPTION
**Pull request by eriadhami**
Closes Issue no. #202

### CODE REVIEW

**Comment by hassanhabib** [comment]
@eriadhami please fix conflicts here as well.

**Comment by eriadhami** [comment]
@hassanhabib done

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (hassanhabib) [inline .vscode/launch.json:4]: @eriadhami this shouldn't be here

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/Taarafo.Core/Taarafo.Core.csproj",
+                "/property:GenerateFullPaths=true",
+                "/consoleloggerparameters:NoSummary"
+            ],
+            "problemMatcher": "$msCompile"
+        },
```
Comment (hassanhabib) [inline .vscode/tasks.json:15]: @eriadhami remove everything under `.vscode/*`

**Pull request closed without merging**